src/app/pages/home-page/offer-list-page/offer-list-page.component.html

```diff
@@ -8,4 +8,7 @@
   >
     <app-offer-item [offer]="offer" class="offer-card"></app-offer-item
   ></a>
+  <div class="offers__text text_s_xl text_c_secondary" *ngIf="!offers?.length">
```

Comment:
even if we use only modificators, we mustn`t skip block basic class. For this case - 'text'.

Reply:
it's my mistake, I'll fix it